pkg/syncer/admissionhandler/cnscsi_admissionhandler.go

```diff
@@ -84,9 +84,15 @@ func (h *CSISupervisorWebhook) Handle(ctx context.Context, req admission.Request
 	log.Debugf("CNS-CSI validation webhook handler called with request: %+v", req)
 
 	resp = admission.Allowed("")
-	if featureGateTKGSHaEnabled {
-		if req.Kind.Kind == "PersistentVolumeClaim" {
-			resp = validatePVCAnnotation(ctx, req)
+	if req.Kind.Kind == "PersistentVolumeClaim" {
+		if featureGateTKGSHaEnabled {
```

Comment:
how do we deal with validation when multiple features are on? 
tomorrow when the new feature comes, do we need to make sure to keep the order of validation?

Reply:
With this code change, it's essentially running validations for the enabled features one after the other,

1. If any of the validations for the enabled features fails, the request is denied. 
2. Only if all the validations for the enabled features succeed, the request will be approved. 

This way, the order of features will not matter. We just need to ensure that when adding a new feature to this list that all features are checked serially and a successful validation from any one of the features doesn't approve the request immediately.